### ISSUE TITLE
LostInstance false-positive

### ISSUE THREAD

**Issue by boris-petrov** [checkstyle-addons/checkstyle-addons#2]
```java
String leftOverText = "";
...
leftOverText += new String(buffer, 0, readBytes);
```

The last line of this example gives a `LostInstance` error which is wrong.

**Comment by tsjensen** [comment]
Dang, thanks for the report! I'll look into it asap.

**Comment by tsjensen** [comment]
Reproduced - it's a bug. 😇 
Stay tuned for a fix. It will probably not be before the weekend, though.

**Comment by boris-petrov** [comment]
No worries, there is no hurry, thanks for the support! :)

**Comment by tsjensen** [comment]
Released as v6.0.1.  
Sync to Maven Central is currently slow, but jcenter should already be good.
